composer.json

```diff
@@ -1,24 +1,24 @@
 {
     "name": "laravel/laravel",
```

Comment:
Maybe change this to `laracasts/council`?

Reply:
Thought about that. Will leave for Jeffrey to decide which to use: 'laracasts/council' or `JeffreyWay/council` or  'council/council' :)